Stylll/Book-A-Meal
# #160249804 implement feedback received from review

#### What does this PR do?
implements feedback received from review session

#### Description of Task to be completed?
- update seeder variable names to be more descriptive
- update test cases to have more descriptive headings
- update meal test suite to test for edge cases

#### How should this be manually tested?
- run `npm run test:server` to see that all tests are passing
- go through seeder files to check the variable names
- go through test suites to check if headings are descriptive
- start the app using `npm run start:app`
- try to create a meal with price of 0.98, the request should not be processed
- try to create a meal with price of 0003000, the meal should be created and the price should be 3000

#### What are the relevant pivotal tracker stories?
#160249804

#### Any background context you want to add?

#### Screenshots
